### DISCUSSION TITLE
A/B testing for notebook results

### DISCUSSION THREAD

**Discussion by SimonHeybrock** [scipp/ess-legacy-v2#92] [General]
- This would be useful outside scipp.
- Create, e.g., 2 conda envs, run notebooks, store tagged results to files
  - Input data would typically be obtained by the notebook using `pooch`. Could inject other files somehow, e.g., to test with a number of different inputs.
- Load files, compare (with tolerance)
- Would be most useful if adding a new "test" would be as simple as adding a line (filename) to a list
- When and where should this run?

**Comment by nvaytet** [comment]
- Add two owners per workflow? one developer for the code itself, one scientist who has the knowledge to approve the scientific results
